src/overlays/actors/ovl_En_Ge3/z_en_ge3.h

```diff
@@ -2,14 +2,54 @@
 #define Z_EN_GE3_H
 
 #include "global.h"
+#include "z64snap.h"
+#include "objects/object_geldb/object_geldb.h"
 
 struct EnGe3;
 
 typedef void (*EnGe3ActionFunc)(struct EnGe3*, PlayState*);
 
+#define GERUDO_AVEIL_GET_PATH(thisx) (((thisx)->params & 0xFC00) >> 10)
+#define GERUDO_AVEIL_GET_EXIT(thisx) (((thisx)->params & 0x1F0) >> 4)
+#define GERUDO_AVEIL_GET_TYPE(thisx) ((thisx)->params & 1)
+
+#define GERUDO_AVEIL_PATH_NONE 0x3F
+
+typedef enum {
+    /* 0 */ GERUDO_AVEIL_TYPE_0, // Unused, just walks forwards
+    /* 1 */ GERUDO_AVEIL_TYPE_AVEILS_CHAMBER
+} GerudoAveilType;
+
+typedef enum {
+    /* -1 */ GERUDO_AVEIL_CSACTION_NONE = -1,
+    /*  0 */ GERUDO_AVEIL_CSACTION_0, // Unused
+    /*  1 */ GERUDO_AVEIL_CSACTION_WAIT,
+    /*  2 */ GERUDO_AVEIL_CSACTION_PAUSE,
+    /*  3 */ GERUDO_AVEIL_CSACTION_INSTRUCT,
+    /*  4 */ GERUDO_AVEIL_CSACTION_DEMAND,
+    /*  5 */ GERUDO_AVEIL_CSACTION_DISMISS,
+    /*  6 */ GERUDO_AVEIL_CSACTION_BEG,
+    /*  7 */ GERUDO_AVEIL_CSACTION_BEEHIVE_RUN_AWAY,
+    /*  8 */ GERUDO_AVEIL_CSACTION_BEEHIVE_VANISH,
+} GerudoAveilCsAction;
```

Comment:
```suggestion
    /*  8 */ GERUDO_AVEIL_CUEID_BEEHIVE_VANISH,
} GerudoAveilCueId;
```
Worth noting this is the new cutscene terminology